web/src/main/webapp/WEB-INF/views/histopathLandingPage.jsp

```diff
@@ -210,7 +210,7 @@
             <tbody>
             <c:forEach var="arow" items="${rows}" varStatus="status">
                 <tr>
-                    <td>${geneSymbols[status.index]}</td>
+                    <td class="cell">${geneSymbols[status.index]}</td>
```

Comment:
To preserve legacy behaviour (although I'm not sure if linking to a page like this: https://www.mousephenotype.org/data/histopath/1500009L16Rik?anatomy="Gene" is correct)

Reply:
That seems to be working fine. We need to review this whole story when we re-implement the histopath pages in the new website.